dockerhub grafana/loki:latest will not start

using Ubuntu 20.04
docker version is 19.03.8

steps to create:
docker pull grafana/loki:latest

docker run grafana/loki:latest

immeidately panics with below error:

level=info ts=2020-07-30T05:37:56.84571017Z caller=main.go:102 msg="Starting Loki" version="(version=master-2a596a7, branch=master, revision=2a596a77)"
level=info ts=2020-07-30T05:37:56.847147865Z caller=server.go:194 http=[::]:3100 grpc=[::]:9095 msg="server listening on addresses"
level=info ts=2020-07-30T05:37:56.848321677Z caller=worker.go:58 msg="no address specified, not starting worker"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1bcca3b]

goroutine 1 [running]:
github.com/cortexproject/cortex/pkg/util/runtimeconfig.(*Manager).GetConfig(0x0, 0x0, 0x0)
	/src/loki/vendor/github.com/cortexproject/cortex/pkg/util/runtimeconfig/manager.go:206 +0x3b
github.com/grafana/loki/pkg/loki.tenantLimitsFromRuntimeConfig.func1(0x0, 0x0, 0xc000984010)
	/src/loki/pkg/loki/runtime_config.go:36 +0x2e
github.com/grafana/loki/pkg/util/validation.(*Overrides).getOverridesForUser(...)
	/src/loki/pkg/util/validation/limits.go:254
github.com/grafana/loki/pkg/util/validation.(*Overrides).IngestionRateStrategy(0xc00085f110, 0xc000984010, 0x0)
	/src/loki/pkg/util/validation/limits.go:139 +0x55
github.com/grafana/loki/pkg/distributor.New(0x27e85f9, 0x6, 0x27f268f, 0xb, 0x27f96ea, 0xe, 0x0, 0x0, 0x4a817c800, 0x0, ...)
	/src/loki/pkg/distributor/distributor.go:110 +0xa4
github.com/grafana/loki/pkg/loki.(*Loki).initDistributor(0xc00020a800, 0x2e4f700, 0xc000687140, 0xc000a7e6c0, 0xc000ba4820)
	/src/loki/pkg/loki/modules.go:132 +0x1c4
github.com/cortexproject/cortex/pkg/util/modules.(*Manager).InitModuleServices(0xc00063cdc0, 0x27e4d3e, 0x3, 0x2e48c80, 0xc0000a9800, 0xc0000a9800)
	/src/loki/vendor/github.com/cortexproject/cortex/pkg/util/modules/modules.go:87 +0x26f
github.com/grafana/loki/pkg/loki.(*Loki).Run(0xc00020a800, 0xc0006868c0, 0x4)
	/src/loki/pkg/loki/loki.go:178 +0x5b
main.main()
	/src/loki/cmd/loki/main.go:104 +0x644

--- a/pkg/loki/runtime_config.go
+++ b/pkg/loki/runtime_config.go
@@ -32,6 +32,9 @@ func loadRuntimeConfig(r io.Reader) (interface{}, error) {
 }
 
 func tenantLimitsFromRuntimeConfig(c *runtimeconfig.Manager) validation.TenantLimits {
+	if c == nil {
+		return nil
+	}
 	return func(userID string) *validation.Limits {
 		cfg, ok := c.GetConfig().(*runtimeConfigValues)
 		if !ok || cfg == nil {
@@ -43,6 +46,9 @@ func tenantLimitsFromRuntimeConfig(c *runtimeconfig.Manager) validation.TenantLi
 }
 
 func multiClientRuntimeConfigChannel(manager *runtimeconfig.Manager) func() <-chan kv.MultiRuntimeConfig {
+	if manager == nil {
+		return nil
+	}
 	// returns function that can be used in MultiConfig.ConfigProvider
 	return func() <-chan kv.MultiRuntimeConfig {
 		outCh := make(chan kv.MultiRuntimeConfig, 1)

